dark/reads.py

```diff
@@ -159,8 +159,9 @@ def checkAlphabet(self, count=10):
         corresponds to its readClass. For AA reads, more testing is done in
         dark.Read.AARead.checkAlphabet.
 
-        @param count: A C{int} of how much of the start of the sequence should
-            be considered.
+        @param count: An C{int}, indicating how many bases at the start of
+            the sequence should be considered. If C{None}, all bases are
+            checked.
```

Comment:
This is not true for amino acids. Can you say something like `indicating how many bases or amino acids at the start of...`?